lib/t/news.ex

```diff
@@ -14,12 +14,177 @@ defmodule T.News do
           "background" => %{"color" => "#F97EB9"},
           "labels" => [
             %{
-              "size" => [247.3, 44],
-              "value" => dgettext("news", "Мы стали совсем другими"),
-              "center" => [142.33, 317.83]
+              "zoom" => 1.0,
+              "value" => dgettext("news", "Привет! 👋"),
+              "position" => [24.0, 180.0],
+              "corner_radius" => 0,
+              "alignment" => 0
+            },
+            %{
+              "zoom" => 1.0,
+              "value" =>
+                dgettext("news", "Это новый Since -\nпространство для\nинтересных людей."),
```

Comment:
did old Since have a different mission? It's not clear that it did.

Reply:
yeah, the key difference in this pivot in audio -> contacts, should articulate it better